org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/splitting/Splitting.java

```diff
@@ -124,7 +124,7 @@ public boolean checkValidTopology (TTopologyTemplate topologyTemplate) {
 			if (!transitiveAndDirectSuccessors.get(node).isEmpty()) {
 				for (TNodeTemplate successor : transitiveAndDirectSuccessors.get(node)) {
 					if (ModelUtilities.getTargetLabel(successor).isPresent() && ModelUtilities.getTargetLabel(node).isPresent()
-						&& !ModelUtilities.getTargetLabel(node).equals(ModelUtilities.getTargetLabel(successor))) {
+						&& !ModelUtilities.getTargetLabel(node).get().toLowerCase().equals(ModelUtilities.getTargetLabel(successor).get().toLowerCase())) {
```

Comment:
Is it possible to use [equalsIgnoreCase](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#equalsIgnoreCase(java.lang.String))?